src/selectors/movie.js

```diff
@@ -0,0 +1,3 @@
+export default (movies, id) => {
```

Comment:
I can't see benefit of having multiple selectors files while one can be easier to deal with.

A nice adding to your selectors is a dependency called `reselect` it can make your life easier and your components more performant because of how reselect works.

And just one reminder:

You selector should be used to hide the complexity of your state, not just as utility functions. I would rewrite this selector as:

```javascript
import { createSelector } from 'reselect';

export const getMovies = (state) => state.movies;
export const getMovieById = createSelector(
    getMovies,
    (_state, ownProps) => ownProps.movieId,
    (movies, id) => movies[id]
);

```

Then in your component/container you will use it as:

```javascript
// id is a prop of your component and is set while rendering this component
const mapStateToProps = (state, ownProps) => ({
    movie: getMovieById(state, ownProps),
});
```

If your state can change without any need for updating your component and also having some memoization to avoid recalculations while using `reselect`. It will be more evident the power of this memoization with your sorting function